guides/INSTALLING.md

```diff
@@ -100,53 +146,98 @@ Unless otherwise stated, services are run using `$ make debug`
 * [dp-frontend-dataset-controller](https://github.com/ONSdigital/dp-frontend-dataset-controller)
   - `git clone git@github.com:ONSdigital/dp-frontend-dataset-controller`
 
+The website will be available at http://localhost:22000
```

Comment:
I am questioning the value of the start up instructions including steps to set up and run apps in _web_ mode. Developers almost always run the stack in publishing mode - **it very rare to run it as web only.**

I think we can omit web set up from this guide. I think it makes a confusing set up longer and more complicated than it needs to be.